src/style.css

```diff
@@ -1 +1,80 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.navigation {
+  font-family: "Roboto", sans-serif;
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation__logo {
+  align-self: center;
+  justify-content: center;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
```

Comment:
```suggestion
  width: 40px;
  height: 40px;
```

Reply:
you don`t need these properties here
you don`t have position and display on logo, or container